idb/postgres_migrations.go

```diff
@@ -59,38 +73,54 @@ func (db *PostgresIndexerDb) runAvailableMigrations(migrationStateJson string) (
 		}
 	}
 
-	if state.NextMigration >= len(migrations) {
-		return nil
-	}
-	if anyBlockers(state.NextMigration) {
-		// run migrations synchronously
-		log.Print("Indexer startup will be delayed until database migration completes")
-		return db.runMigrationState(&state, true)
-	} else {
-		log.Print("database migrations running in background...")
-		go db.runMigrationState(&state, false)
-		return nil
-	}
-}
+	abortChan := make(chan error)
+	blockChan := make(chan struct{})
 
-func (db *PostgresIndexerDb) runMigrationState(state *MigrationState, blocking bool) (err error) {
```

Comment:
The diff view probably wont be very useful with this function, probably best to look at just the new task queue I've implemented.